content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -146,7 +151,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    sudo apt-get install -y kubectl
    ```
 {{< note >}}
-In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
+In releases older than Debian 12 and Ubuntu 21.10 `/etc/apt/keyrings` does not exist by default.
```

Comment:
Why should we make this change? (did we check that Ubuntu 21.04 is the newest release that doesn't have that directory?)